### PR TITLE
Windows linux AI, and ICO updates to c4

### PR DESCRIPTION
**Pull request by daveshrestha-intel**
### **User description**
This updates both main.tf and readme files for AI and ICO examples for C4 instance

___

### **PR Type**
enhancement, documentation

___

### **Description**
- Updated machine types across multiple examples to use `c4` series for improved performance.
- Removed local `versions.tf` files to resolve version conflicts.
- Updated documentation to reflect the use of 5th Gen Xeon processors.
- Adjusted `recommendedType` settings in ICO examples to align with new machine types.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement</strong></td><td><details><summary>7 files</summary><table>
<tr>
  <td>
    <details>
      <summary><strong>main.tf</strong><dd><code>Update machine type to c4-standard-32 in FastChat example</code></dd></summary>
<hr>

examples/gcp-linux-fastchat-simple/main.tf

- Updated `machine_type` to `c4-standard-32`.

</details>

  </td>
  <td><a href="https://github.com/intel/terraform-intel-gcp-vm/pull/48/files#diff-62be20b44cab1eec2122e377b9fd982542e24cef42a7d61fb09d0e224dfa7d81">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>main.tf</strong><dd><code>Update machine type to c4-standard-32 in Stable Diffusion example</code></dd></summary>
<hr>

examples/gcp-linux-stable-diffusion/main.tf

- Changed `machine_type` to `c4-standard-32`.

</details>

  </td>
  <td><a href="https://github.com/intel/terraform-intel-gcp-vm/pull/48/files#diff-5beb6467542e9171653eae90d0deae65a560da90cec8c2a42cf111b8909281e6">+1/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>main.tf</strong><dd><code>Update machine type to c4-standard-4 in TDX VM example</code>&nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

examples/gcp-linux-tdx-vm/main.tf

- Modified `machine_type` to `c4-standard-4`.

</details>

  </td>
  <td><a href="https://github.com/intel/terraform-intel-gcp-vm/pull/48/files#diff-f7262809ee45b48d9d8b7c294e8a809102b174666f0d1069845720ab59a5750e">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>variables.tf</strong><dd><code>Update recommendedType to c4-standard-4 in Linux VM ICO example</code></dd></summary>
<hr>

examples/gcp-linux-vm-ico-by-densify/variables.tf

- Updated `recommendedType` to `c4-standard-4`.

</details>

  </td>
  <td><a href="https://github.com/intel/terraform-intel-gcp-vm/pull/48/files#diff-44fae723ef00b5d0f0a32caeeb05d0d224224bec6c0517b8dc51685d154d1ebb">+2/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>variables.tf</strong><dd><code>Update recommendedType to c4-standard-4 in Windows VM ICO example</code></dd></summary>
<hr>

examples/gcp-windows-vm-ico-by-densify/variables.tf

- Changed `recommendedType` to `c4-standard-4`.

</details>

  </td>
  <td><a href="https://github.com/intel/terraform-intel-gcp-vm/pull/48/files#diff-5dc5dde664f8ad95045d449c655358ccdcb3bc5ac552f27e5d462f0ca0ccb247">+2/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>densify_recommendations.auto.tfvars</strong><dd><code>Update recommendedType in Linux VM ICO auto.tfvars</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

examples/gcp-linux-vm-ico-by-densify/densify_recommendations.auto.tfvars

- Changed `recommendedType` to `c4-standard-4`.

</details>

  </td>
  <td><a href="https://github.com/intel/terraform-intel-gcp-vm/pull/48/files#diff-4a4021a5e6507d67b52a7ab2ad07599e9a04e8e4fc04cc4b885ef536cb860777">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>densify_recommendations.auto.tfvars</strong><dd><code>Update recommendedType in Windows VM ICO auto.tfvars</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

examples/gcp-windows-vm-ico-by-densify/densify_recommendations.auto.tfvars

- Changed `recommendedType` to `c4-standard-4`.

</details>

  </td>
  <td><a href="https://github.com/intel/terraform-intel-gcp-vm/pull/48/files#diff-d779c2f21c7643278f089a6ea3c00d305b0c96b8ce54a694cd277dfba53d20e7">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></details></td></tr><tr><td><strong>Configuration changes</strong></td><td><details><summary>2 files</summary><table>
<tr>
  <td>
    <details>
      <summary><strong>versions.tf</strong><dd><code>Remove local versions.tf file in RHEL VM example</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

examples/gcp-rhel-vm/versions.tf

- Removed local `versions.tf` file.

</details>

  </td>
  <td><a href="https://github.com/intel/terraform-intel-gcp-vm/pull/48/files#diff-514bb715ae6c50cdd524a5f4d4d9245d78bbbddd85a2f724d9ebc791ab275170">+0/-10</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>versions.tf</strong><dd><code>Remove local versions.tf file in Windows VM example</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

examples/gcp-windows-vm/versions.tf

- Deleted local `versions.tf` file.

</details>

  </td>
  <td><a href="https://github.com/intel/terraform-intel-gcp-vm/pull/48/files#diff-edfb16024a169450b4a278057b113b82e2bca6c9989760ceda188e6a0d42e65f">+0/-10</a>&nbsp; &nbsp; </td>

</tr>                    
</table></details></td></tr><tr><td><strong>Documentation</strong></td><td><details><summary>7 files</summary><table>
<tr>
  <td>
    <details>
      <summary><strong>main.tf</strong><dd><code>Correct comment for firewall rules in AI OPEA Codegen example</code></dd></summary>
<hr>

examples/gen-ai-xeon-opea-codegen/main.tf

- Corrected comment for firewall rules.

</details>

  </td>
  <td><a href="https://github.com/intel/terraform-intel-gcp-vm/pull/48/files#diff-d5663b2367d15a6e2b24e0011fad5a42b0cf5caea87b51439e20ec94037496ed">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>README.md</strong><dd><code>Update processor generation in FastChat README</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

examples/gcp-linux-fastchat-simple/README.md

- Updated processor generation to 5th Gen Xeon.

</details>

  </td>
  <td><a href="https://github.com/intel/terraform-intel-gcp-vm/pull/48/files#diff-55b286701bb6338861e4d2947ded46ec07c6e45aed4ad58f85a9bfac52a2e681">+2/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>README.md</strong><dd><code>Update processor generation in Stable Diffusion README</code>&nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

examples/gcp-linux-stable-diffusion/README.md

- Updated processor generation to 5th Gen Xeon.

</details>

  </td>
  <td><a href="https://github.com/intel/terraform-intel-gcp-vm/pull/48/files#diff-30f889a119d8441ad54e64cf4147df00deeb3b7995a956e7bf668d49d1528930">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>README.md</strong><dd><code>Update processor generation in TDX VM README</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

examples/gcp-linux-tdx-vm/README.md

- Updated processor generation to 5th Gen Xeon.

</details>

  </td>
  <td><a href="https://github.com/intel/terraform-intel-gcp-vm/pull/48/files#diff-2f2663c289f6a2f00a9184fbfcad541bacfda4914da24419dcb6097110ca53c9">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>README.md</strong><dd><code>Update recommendedType in Linux VM ICO README</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

examples/gcp-linux-vm-ico-by-densify/README.md

- Updated `recommendedType` to `c4-standard-4`.

</details>

  </td>
  <td><a href="https://github.com/intel/terraform-intel-gcp-vm/pull/48/files#diff-63e1a5c99db381d9f4bfb250566b229fdfe89ed7b042a2d55ec6ac0255163442">+2/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>README.md</strong><dd><code>Update processor generation in RHEL VM README</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

examples/gcp-rhel-vm/README.md

- Updated processor generation to 5th Gen Xeon.

</details>

  </td>
  <td><a href="https://github.com/intel/terraform-intel-gcp-vm/pull/48/files#diff-4c99f410b654610239afb384f8545b78772476f61f1fddc9af4c43560ef3856b">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>README.md</strong><dd><code>Update recommendedType in Windows VM ICO README</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

examples/gcp-windows-vm-ico-by-densify/README.md

- Updated `recommendedType` to `c4-standard-4`.

</details>

  </td>
  <td><a href="https://github.com/intel/terraform-intel-gcp-vm/pull/48/files#diff-464d7f441c9bddbee7192840a76da072280bad8fc6d500f7f0505fa646a62c94">+2/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></details></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

File: examples/gcp-windows-vm/versions.tf
Comment (lucasmelogithub) [inline examples/gcp-windows-vm/versions.tf:1]: Did you mean to delete this file? Just checking.

File: examples/gcp-rhel-vm/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 ## Intel Red Hat Enterprise Linux GCP VM Example
 
-This module creates a Red Hat Enterprise Linux (RHEL) VM on the Intel Sapphire Rapids CPU. The virtual machine is created on an Intel Sapphire Rapids c3-standard-4 by default.
+This module creates a Red Hat Enterprise Linux (RHEL) VM on the Intel 5th Gen Emerald Rapids CPU. The virtual machine is created on an Intel Sapphire Rapids c3-standard-4 by default.
```
Comment (lucasmelogithub) [inline examples/gcp-rhel-vm/README.md:11]: Correct this "Intel Sapphire Rapids c3-standard-4"

Comment (daveshrestha-intel) [reply examples/gcp-rhel-vm/README.md:11]: It's should say c4-standard-4 by default- Emerald Rapids.

**Review by lucasmelogithub — APPROVED** [review]
LGTM

**Pull request merged**